docs/cli/tkn-results_taskrun_list.md

```diff
@@ -0,0 +1,74 @@
+## tkn-results taskrun list
+
+List TaskRuns in a namespace
+
+```
+tkn-results taskrun list [task-name]
+```
+
+### Examples
+
+```
+List all TaskRuns in a namespace 'foo':
+    tkn-results taskrun list -n foo
+
+List all TaskRuns in 'default' namespace:
+    tkn-results taskrun list -n default
```

Comment:
should we skip `-n default` for listing taskrun list from `default` namespace, wdyt?

Reply:
IMO no. I would prefer that to be explicit and not assume any defaults. Again, this is only my opinion.

Reply:
Yeah, if `-n` flag is not used, then it will use the current namespace from the context (not specifically `default`)
I can update the example to avoid any confusion.

Reply:
Done